third_party/tizen/tizen_qemu.py

```diff
@@ -65,10 +65,8 @@
     help=("path to the system data image; "
           "default: $TIZEN_SDK_ROOT/iot-sysdata.img"))
 parser.add_argument(
-    '--image-iso', metavar='IMAGE',
-    help=("path to the ISO image with the runner script; the ISO image "
-          "should have 'CHIP' label and a file named 'runner.sh' at the "
-          "root directory"))
+    '--share', type=str,
+    help="directory to be shared, runner.sh inside will be run if present")
```

Comment:
Maybe more verbose help, something like:

```suggestion
    help=("host directory to share with the guest; if file named 'runner.sh' "
          "is present at the root of that directory, it will be executed "
          "automatically after boot"))
```